jetty-annotations/src/main/java/org/eclipse/jetty/annotations/PostConstructAnnotationHandler.java

```diff
@@ -73,7 +73,7 @@ public void doHandle(Class clazz)
                         return;
 
                     PostConstructCallback callback = new PostConstructCallback();
```

Comment:
As discussed, construction injection is probably better here.    Perhaps fix when merging this to jetty-10